src/random-walk.js

```diff
@@ -52,23 +52,18 @@ class RandomWalk {
           runningHandle._timeoutId = null
```

Comment:
It seems like we're no longer using `runningHandle._onCancel`. Maybe the code could be simplified by replacing `this.runningHandle` with `this.timeoutId`, and making `runPeriodically` a class method?

Reply:
Good call, I will review the code there.